argus/util.py

```diff
@@ -50,6 +51,8 @@
 DEFAULT_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
 DEFAULT_LOG_FILE = 'argus.log'
 
+RETRY_FOREVER = (winrm.exceptions.WinRMTransportError,)
```

Comment:
Can we drop the support of retrying forever? Just use a big timeout with a good number of retries when you know that the function runs when the instance reboots.